raspberry/bzzz/sensors/evo_time_of_flight.py

```diff
@@ -0,0 +1,250 @@
+import serial
+import crcmod.predefined  # To install: pip install crcmod
+import numpy as np
+from threading import Thread, Lock
+import abc
+import time
+import datetime
+import csv
+ 
+ 
+class DataProcessor(abc.ABC):
```

Comment:
To avoid code duplication, these should be moved to a separate file and then imported where necessary

Reply:
done

Reply:
done